app/assets/javascripts/educator/EducatorPage.test.js

```diff
@@ -1,24 +1,21 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import EducatorPage from './EducatorPage';
 import SpecSugar from '../../../../spec/javascripts/support/spec_sugar.jsx';
 import {withDefaultNowContext} from '../../../../spec/javascripts/support/NowContainer';
 import fetchMock from 'fetch-mock/es5/client';
+import EducatorPage from './EducatorPage';
 import educatorsViewJson from '../../../../spec/javascripts/fixtures/educatorsViewJson';
-import homeFeedJson from '../../../../spec/javascripts/fixtures/home_feed_json';
-import unsupportedLowGradesJson from '../../../../spec/javascripts/fixtures/home_unsupported_low_grades_json';
+
 
 function testProps() {
   return {
     educatorId: 101
   };
 }
 
+jest.mock('../home/HomePage');
```

Comment:
Using this in place of Enzyme's `shallow` renderer since it lets all the code run like normal and just mocks this one dependency (which requires other test setup because of the HTTP requests, we don't need to test all that here).